### PR TITLE
docs: update readme.md with instructions

### LINKED ISSUE #29: Readme Update
The readme file of this project is the default for now. We need to add information about the tools we are using for this project and list the contribution requirements and standards

### PR DESCRIPTION
**Pull request by divyanshugarg36**
Update readme with all the required instructions related to the usage of this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,72 @@
+# BRL-CAD Web 2022
+
+[![IRC](https://img.shields.io/badge/chat-on%20irc%20channel-green.svg)](http://webchat.freenode.net/?channels=#brlcad)
+[![IRC](https://img.shields.io/badge/join-mailing%20list-green.svg)](https://lists.sourceforge.net/lists/listinfo/brlcad-devel)
+
```
Comment (theamanjs) [inline README.md:6]: You can probably add zulip here as well.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,72 @@
...
+Clone this repo
+
+```sh
+git clone https://github.com/divyanshugarg36/brl-cad-web.git
+```
 
-```bash
+###### Install Nodejs :
+```
+sudo apt-get install nodejs
```
Comment (theamanjs) [inline README.md:36]: Can you add prerequisites for the project as well?

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (divyanshugarg36) [reply README.md:6]: Added

File: README.md (hunk 2 above)
Comment (divyanshugarg36) [reply README.md:36]: Added

**Review by theamanjs — APPROVED** [review]

**Pull request merged**